keps/sig-cluster-lifecycle/kubeadm/20191125-remove-clusterstatus-from-kubeadm-config.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Remove ClusterStatus from kubeadm-config
+authors:
+  - "@fabriziopandini"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-cluster-lifecycle
+reviewers:
+  - "@neolit123"
+  - "@rosti"
+  - "@ereslibre"
+  - "@ncdc"
+approvers:
+  - "@timothysc"
+editor: "@fabriziopandini"
+creation-date: 2019-11-25
+last-updated: 2019-11-25
+status: implementable
```

Comment:
should be provisional as a start?

Reply:
If there is agreement, we can get this merged as implementable and start immediately the implementation (it already happened in the past)

Reply:
SGTM